docs/docs/explanations/advanced-profiles-settings.md

```diff
@@ -0,0 +1,135 @@
+
+`kubespawner_override` field to define behavior as per the [KubeSpawner](https://jupyterhub-kubespawner.readthedocs.io/en/latest/spawner.html) API.
+
+It is possible to control which users have access to which JupyterLab profiles. Each profile has a field named `access` which can be set to `all` (default if omitted), `yaml`, or
+`keycloak`.
+
+- `all` means every user will have access to the profile.
+- `yaml` means that access is restricted to anyone with their username in the `users` field of the profile or who belongs to a group named in the `groups` field.
+- `keycloak` means that access is restricted to any user who in Keycloak has either their group(s) or user with the attribute `jupyterlab_profiles` containing this profile name. For
+  example, if the user is in a Keycloak group named `developers` which has an attribute `jupyterlab_profiles` set to `Large Instance`, they will have access to the Large Instance
+  profile. To specify multiple profiles for one group (or user) delimit their names using `##` - for example, `Large Instance##Another Instance`.
+
+When configuring the memory and CPUs for profiles, there are some important considerations to make. Two important terms to understand are:
+
+- `limit`: the absolute max memory that a given pod can consume. If a process within the pod consumes more than the `limit` memory the linux OS will kill the process. Limit is not
+  used for scheduling purposes with kubernetes.
+- `guarantee`: is the amount of memory the kubernetes scheduler uses to place a given pod. In general the `guarantee` will be less than the limit. Often times the node itself has
+  less available memory than the node specification. See this [guide from digital ocean](https://docs.digitalocean.com/products/kubernetes/details/limits/#allocatable-memory) which is generally
+  applicable to other clouds.
+
+For example if a node has 8 GB of ram and 2 CPUs you should guarantee/schedule roughly 75% and follow the digital ocean guide linked above, e.g. 1.5 CPU guaranteed and 5.5 GB guaranteed.
+
+### JupyterLab Profile Node Selectors
+
+A common operation is to target jupyterlab profiles to specific node labels. In order to target a specific node groups add the following.
+This example shows a GKE node groups with name `user-large`.
+Other cloud providers will have different node labels.
+
+```yaml
+### JupyterLab Profiles ###
+profiles:
+  jupyterlab:
+    - display_name: Small Instance
+      ...
+      kubespawner_override:
+        ...
+        node_selector:
+          "cloud.google.com/gke-nodepool": "user-large"
+        ...
+```
+
+### Specifying GPU/Accelerator Requirements
+
+If you want to ensure that you have GPU resources use the following annotations.
+
+```yaml
+### JupyterLab Profiles ###
+profiles:
+  jupyterlab:
+    - display_name: Small Instance
+      ...
+      kubespawner_override:
+        ...
+        extra_resource_limits:
+          nvidia.com/gpu: 1
+        ...
+```
+
+## Dask profiles
```

Comment:
We need to add a cross reference in the Dask docs back to this config. I made a note of it [here](https://github.com/nebari-dev/nebari-docs/pull/250/files?notification_referrer_id=NT_kwDOAO7wZrM1MTk2NzAwNjcxOjE1NjU5MTEw&notifications_query=repo%3Anebari-dev%2Fnebari-docs#discussion_r1084319185) .